src/_actions/category.actions.js

```diff
@@ -5,24 +5,61 @@ import axios from 'axios';
 import constants from '../_constants';
 
 const {
+  LOADING_CATEGORIES,
   LOAD_CATEGORIES_SUCCESS,
   LOAD_CATEGORY_FAILURE,
   CREATE_CATEGORY_SUCCESS,
   CREATE_CATEGORY_FAILURE,
   UPDATE_TOAST_MESSAGE_CONTENT
 } = constants;
 
+export const loadCategoriesDropdown = pageNumber => (dispatch) => {
+  dispatch({ type: LOADING_CATEGORIES });
+
+  return axios.get(`asset-categories/?page=${pageNumber}`)
+    .then((response) => {
+      const pageLimit = Math.ceil(response.data.count / 10);
+
+      if (pageLimit > 1) {
+        dispatch(loadCategoriesSuccess(response.data));
+        while (pageNumber < pageLimit) {
+          pageNumber += 1;
+
+          if (response.data.next !== '') {
+            axios.get(`asset-categories/?page=${pageNumber}`)
```

Comment:
Are you making 2 get requests because the endpoint hasn't been built yet?

Reply:
Yes, this is what Eric requested that we do for the time being